README.md

```diff
@@ -27,27 +47,36 @@ You will need the following tools to clone, build, and run Glimesh:
 - **nodejs / npm**: Front-end package management
 - **inotify-tools**: Filesystem monitoring dependencies for developer convenience (watching changes)
 
-On modern versions of Ubuntu, you can install these packages with the following command:
-
-```sh
-sudo apt install git esl-erlang elixir postgresql npm inotify-tools
-```
+You may need to translate these exact dependencies into their appropriate names for your OS distribution.
 
-Other distributions likely have packages available for these tools as well.
-
-## Cloning
+### Cloning
 
 To clone a local copy of Glimesh.tv, run
 
 ```sh
-git clone git@github.com:Glimesh/glimesh.tv.git
+git clone https://github.com/Glimesh/glimesh.tv.git
+# or if you have permissions to the repository
+# git clone git@github.com:Glimesh/glimesh.tv.git
 ```
 
 **NOTE**: If you are using WSL2, ensure you are cloning inside of your WSL2 instance
 (ex. `/home/user/...`) and not inside of a mounted Windows drive (ex. `/mnt/c/Users/...`)
 as this can have a significant negative impact on performance.
 
-## Configuring Postgres
+### Windows Installation
+
+Running Glimesh.tv natively on Windows is not yet understood. However you can run the application very well with WSL2. If you are interested in running Glimesh.tv natively, have a go at us and let us know!
+
+[WSL2](https://docs.microsoft.com/en-us/windows/wsl/install-win10) with Ubuntu 20.04 has been tested successfully for development on Windows.
```

Comment:
i have tested this with the 18.04 version aswell as that is what im running on and it also works

Reply:
Updated, thanks!